plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -527,6 +542,12 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
                   fc_dlog(_trx_successful_trace_log, "[TRX_TRACE] Speculative execution is ACCEPTING tx: ${txid}",
                           ("txid", trx->id()));
                }
+
+               if (_trx_trace_dump_log.is_enabled(fc::log_level::debug)) {
+                   //Dump transaction trace
+                   auto entire_trace = chain_plug->get_entire_trx_trace(std::get<transaction_trace_ptr>(response) );
+                   fc_dlog(_trx_trace_dump_log, "[TRX_TRACE] ${entire_trace}", ("entire_trace", entire_trace));
```

Comment:
Add "Speculative execution is ACCEPTING tx:"

Reply:
Fixed.